groclient/client.py

```diff
@@ -1012,11 +1012,12 @@ def get_df(
 
         Parameters
         ----------
-            show_revisions : boolean, optional
-                False by default, meaning only the latest value for each period. If true, will return
-                all values for a given period, differentiated by the `reporting_date` field.
-            show_available_date : boolean, optional
-                False by default. If true, will return the available date of each data point.
+            reporting_history : boolean, optional
```

Comment:
(same for `get_data_points` below)
I think here it's better to elaborate that this parameter controls the source reporting history 
Also, I found here "value " is a bit confusing

maybe something like this? 
```
reporting_history: boolean, optional
False by default. If true, will return all reporting history from the source. 
```